twisted/scripts/_twistd_unix.py

```diff
@@ -198,6 +200,38 @@ def preApplication(self):
         self.oldstderr = sys.stderr
 
 
+    def _formatException(self, exception):
```

Comment:
This isn't strictly related to the problem described in the ticket, but without it twistd won't work correctly on Python 3.